testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/ha/AbstractMessagingHATestCase.java

```diff
@@ -151,22 +150,24 @@ protected static void checkHornetQServerStartedAndActiveAttributes(JMSOperations
     protected static InitialContext createJNDIContextFromServer1() throws NamingException {
         final Properties env = new Properties();
         env.put(Context.INITIAL_CONTEXT_FACTORY, "org.jboss.naming.remote.client.InitialContextFactory");
-        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://127.0.0.1:8080"));
+        String ipAdddress = TestSuiteEnvironment.formatPossibleIpv6Address(System.getProperty("node0", "127.0.0.1"));
+        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://" + ipAdddress + ":8080"));
         env.put(Context.SECURITY_PRINCIPAL, "guest");
         env.put(Context.SECURITY_CREDENTIALS, "guest");
         return new InitialContext(env);
     }
 
-    protected static  InitialContext createJNDIContextFromServer2() throws NamingException {
+    protected static InitialContext createJNDIContextFromServer2() throws NamingException {
         final Properties env = new Properties();
         env.put(Context.INITIAL_CONTEXT_FACTORY, "org.jboss.naming.remote.client.InitialContextFactory");
-        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://127.0.0.1:8180"));
+        String ipAdddress = TestSuiteEnvironment.formatPossibleIpv6Address(System.getProperty("node1", "127.0.0.1"));
+        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://" + ipAdddress + ":8180"));
```

Comment:
This violates RFC, [ and ] are missing.

Reply:
[] are added by TestSuiteEnvironment.formatPossibleIpv6Address